dev-python/traits/traits-4.5.0-r1.ebuild

```diff
@@ -2,9 +2,9 @@
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
 
-PYTHON_COMPAT=(python{2_7,3_3})
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
```

Comment:
bring back 3.3, even though its on its way out. Just to prevent depgraph breakages for those still using it.